fe/fe-core/src/main/java/org/apache/doris/nereids/CascadesContext.java

```diff
@@ -388,19 +389,24 @@ private Set<UnboundRelation> extractUnboundRelationFromCTE(LogicalCTE<?> cte) {
         return relations;
     }
 
-    private Table getTable(UnboundRelation unboundRelation) {
+    private TableIf getTable(UnboundRelation unboundRelation) {
         List<String> nameParts = unboundRelation.getNameParts();
         switch (nameParts.size()) {
             case 1: { // table
+                String ctlName = getConnectContext().getEnv().getCurrentCatalog().getName();
```

Comment:
Do we need to check if `current catalog` is set in `connection context`? Otherwise we may got NPE here

Reply:
Same issue for dbName